examples/platform/silabs/efr32/wf200/host_if.cpp

```diff
@@ -98,6 +98,8 @@ bool hasNotifiedIPV4             = false;
 bool hasNotifiedWifiConnectivity = false;
 static uint8_t retryJoin         = 0;
 bool retryInProgress             = false;
+bool is_commissioned             = false;
```

Comment:
Matter SDK provides an API about the commissioning status.
Looks like you want to maintaing